ddtrace/tracer/sqlcomment_test.go

```diff
@@ -20,53 +20,69 @@ import (
+			expectedSpanIDGen:  true,
+			expectedExtractErr: nil,
 		},
 		{
-			name:              "service",
-			query:             "SELECT * from FOO",
-			mode:              DBMPropagationModeService,
-			injectSpan:        true,
-			expectedQuery:     "/*dddbs='whiskey-db',dde='test-env',ddps='whiskey-service%20%21%23%24%25%26%27%28%29%2A%2B%2C%2F%3A%3B%3D%3F%40%5B%5D',ddpv='1.0.0'*/ SELECT * from FOO",
-			expectedSpanIDGen: false,
+			name:               "service",
+			query:              "SELECT * from FOO",
+			mode:               DBMPropagationModeService,
+			injectSpan:         true,
+			expectedQuery:      "/*dddbs='whiskey-db',dde='test-env',ddps='whiskey-service%20%21%23%24%25%26%27%28%29%2A%2B%2C%2F%3A%3B%3D%3F%40%5B%5D',ddpv='1.0.0'*/ SELECT * from FOO",
+			expectedSpanIDGen:  false,
+			expectedExtractErr: ErrSpanContextNotFound,
 		},
 		{
-			name:              "no-trace",
-			query:             "SELECT * from FOO",
-			mode:              DBMPropagationModeFull,
-			expectedQuery:     "/*dddbs='whiskey-db',ddps='whiskey-service%20%21%23%24%25%26%27%28%29%2A%2B%2C%2F%3A%3B%3D%3F%40%5B%5D',traceparent='00-0000000000000000<span_id>-<span_id>-00'*/ SELECT * from FOO",
-			expectedSpanIDGen: true,
+			name:               "no-trace",
+			query:              "SELECT * from FOO",
+			mode:               DBMPropagationModeFull,
+			expectedQuery:      "/*dddbs='whiskey-db',ddps='whiskey-service%20%21%23%24%25%26%27%28%29%2A%2B%2C%2F%3A%3B%3D%3F%40%5B%5D',traceparent='00-0000000000000000<span_id>-<span_id>-00'*/ SELECT * from FOO",
+			expectedSpanIDGen:  true,
+			expectedExtractErr: nil,
 		},
 		{
-			name:              "no-query",
-			query:             "",
-			mode:              DBMPropagationModeFull,
-			injectSpan:        true,
-			expectedQuery:     "/*dddbs='whiskey-db',dde='test-env',ddps='whiskey-service%20%21%23%24%25%26%27%28%29%2A%2B%2C%2F%3A%3B%3D%3F%40%5B%5D',ddpv='1.0.0',traceparent='00-0000000000000000000000000000000a-<span_id>-00'*/",
-			expectedSpanIDGen: true,
+			name:               "no-query",
+			query:              "",
+			mode:               DBMPropagationModeFull,
+			injectSpan:         true,
+			expectedQuery:      "/*dddbs='whiskey-db',dde='test-env',ddps='whiskey-service%20%21%23%24%25%26%27%28%29%2A%2B%2C%2F%3A%3B%3D%3F%40%5B%5D',ddpv='1.0.0',traceparent='00-0000000000000000000000000000000a-<span_id>-00'*/",
+			expectedSpanIDGen:  true,
+			expectedExtractErr: nil,
 		},
 		{
-			name:              "commented",
-			query:             "SELECT * from FOO -- test query",
-			mode:              DBMPropagationModeFull,
-			injectSpan:        true,
-			samplingPriority:  1,
-			expectedQuery:     "/*dddbs='whiskey-db',dde='test-env',ddps='whiskey-service%20%21%23%24%25%26%27%28%29%2A%2B%2C%2F%3A%3B%3D%3F%40%5B%5D',ddpv='1.0.0',traceparent='00-0000000000000000000000000000000a-<span_id>-01'*/ SELECT * from FOO -- test query",
-			expectedSpanIDGen: true,
+			name:               "commented",
```

Comment:
Maybe we don't want to support this but it might make sense to have a test case where the query includes a few comments in the `/* comment */` format since there could be more than a tracer-injected comment (if the application is already using comments in their queries).